<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZSet_LoopItemWid</class>
 <widget class="QWidget" name="ZSet_LoopItemWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>回路电流最大值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_1" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_6" native="true">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
